android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why are there all of these Android changes? Also this looks like it is removing our Android adaptive icon. I don't think we wanna do that.